src/utils/config-helper.ts

```diff
@@ -84,10 +77,14 @@ export async function fetchConfig (): Promise<any> {
   (<any>window).addressCompleteKey = addressCompleteKey
   console.info('Set Address Complete Key.')
 
-  const ldClientId: string = response.data['LD_CLIENT_ID'];
+  const ldClientId: string = response.data['BUSINESS_CREATE_LD_CLIENT_ID'];
   (<any>window).ldClientId = ldClientId
   console.info('Set Launch Darkly Client ID.')
 
+  const sentryEnable = response.data['SENTRY_ENABLE'];
+  (<any>window).sentryEnable = sentryEnable
+  console.log('Set Sentry DSN Enable.')
```

Comment:
What do you mean? We want to keep the sentry dsn value in 1password. However, I added a switch to control it whether we want to use it or not.

Reply:
I'm referring to the console.log statement.

It might be useful to display whether the value is True or False. Or maybe we don't want to display anything. What do you think?

Reply:
(My preference would be to display the value.)

Reply:
oh, ok. I will remove the log.

Reply:
> (My preference would be to display the value.)